lib/cli.rb

```diff
@@ -246,14 +246,27 @@ def self.supports_filtering(command)
     end
   end
 
-  def self.check_port_validity(port)
+  def self.check_port_validity(port, config_hint = false)
```

Comment:
Assignment Branch Condition size for check_port_validity is too high. [18.38/15]